### DISCUSSION TITLE
Last-minute Session Participant Gap-filling

### DISCUSSION THREAD

**Discussion by bcholmes** [LVerhulst4321/PlanZ#44] [General]
How do your cons handle the case where the preliminary schedule is done, but you have a number of sessions that could use a few more participants? Have you done anything in your implementations to help support that?

**Comment by LVerhulst4321** [comment]
At Capricon, the program team would look at reports that show low counts of panelists assigned to sessions and then send out emails to the participants to see if there was any interest in being added to the sessions. Emails were not sent out via Zambia, but we might try to change that for next year's Cap and use PlanZ.
Identifying the panels that needed more panelists is a little tricky as we put the film schedule and the events schedule and everything into Zambia and not all sessions need a full set of panelists. Using Division usually works though.
